UVicMartletplace/martletplace
# Arun/add-new-dataset/fix-refactor-recommender

# Description
This PR adds a new dataset of 50000 items across a variety of categories (with images). 

A new Recommender release was uploaded to GitHub as well, and the Recommender version has been updated.

Parts of the recommender were refactored (API return values). And fixed (recommender not outputting). 

Closes #508 #509 

## How to Test
Spin up the app and look at the new listings, try clicking/searching, and see how it updates the recommender.


## Checklist
